.github/workflows/integration_test.yml

```diff
@@ -8,41 +8,31 @@ jobs:
     runs-on: ubuntu-20.04
     strategy:
       matrix:
-        kube_provider: [kind]
+        cluster_provider: [kind,microshift]
     steps:
       - uses: actions/checkout@v3
       - uses: actions/setup-go@v3
         with:
           go-version: 1.18
 
-      - name: use kepler action for kind cluster build
-        uses: sustainable-computing-io/kepler-action@v0.0.0
+      - name: use kepler action for cluster build
+        uses: sustainable-computing-io/kepler-action@main
```

Comment:
@rootfs, @vprashar2929  for dev branch, we should keep this to main, for release we should use a release branch. Hence before we bump up version for kepler-action and local-dev-cluster, we should keep it as v0.0.0 and keep this PR open for review.

Reply:
hence I will suggestion as bump up version in another PR.

Reply:
Sure bumping up the version in another PR sounds good. Till that time I will keep it as v0.0.0